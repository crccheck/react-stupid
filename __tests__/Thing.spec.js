const React = require('react');
const renderer = require('react-test-renderer');
const Thing = require('../Thing');

test('Thing renders', () => {
  const component = renderer.create(
    <Thing />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
