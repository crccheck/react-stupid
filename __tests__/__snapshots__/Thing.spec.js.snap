// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thing renders 1`] = `
<div
  className="Thing"
>
  I am a React Thing
</div>
`;
